databases/week2/homework_part1.sql

```diff
@@ -0,0 +1,48 @@
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO 
+    task(id, title, description, created, updated, due_date, status_id, user_id)
+VALUES 
+    (38, "Fly to the Mars", "To take a look", "2022-12-18 13:17:17",  "2022-12-18 13:17:17", "2025-12-18 13:17:17", 1, 11);
+
+-- Change the title of a task
+
+UPDATE
+    task
+SET
+    title = "Fly back home"
+WHERE
+    title like "%Mars%";
+
+-- Change a task due date
+UPDATE
+    task
+SET
+    due_date = "2024-12-18 13:17:17"
+WHERE
+    due_date > "2024-12-18";
+
+-- Change a task status
+UPDATE
+    task
+SET
+    status_id = 2
+WHERE
+    id = 36;
+
+-- Mark a task as complete
+UPDATE
+    task
+SET
+    status_id = 3
+WHERE id = 36;
+
+-- Delete a task
+DELETE
```

Comment:
this works, and its quite clever to solve the task, but keep in mind that deletions are quite "serious" operations, so its good to have specific conditions for what you want to delete exactly.

Reply:
And also her. Task was, that we self chose a modifying entities.